theme6/source/stylesheets/site.css.scss

```diff
@@ -1,24 +1,184 @@
-body {
-  background-color: #fbc547;
-  color: #333;
-  font-family: -apple-system, BlinkMacSystemFont, "Avenir Next", "Avenir",
-    "Segoe UI", "Lucida Grande", "Helvetica Neue", "Helvetica", "Fira Sans",
-    "Roboto", "Noto", "Droid Sans", "Cantarell", "Oxygen", "Ubuntu",
-    "Franklin Gothic Medium", "Century Gothic", "Liberation Sans", sans-serif;
-  padding: 18vh 1rem;
+/*==========Stylesheet DESKTOP========*/
+footer {
+  bottom: 0;
+  display: inline;
+  width: 100%;
+}
+
+.information {
+  justify-content: center;
+}
+
+li {
+  display: inline;
+}
+
+.socialmedia {
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+}
+
+.sm {
+  height: 35px;
+  width: 93px;
+  border: 2px solid #72808C;
+  border-radius: 3px;
+}
+
+#fb {
+  background: url("../images/fb.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+  margin-right: 23px;
+}
+
+#twt {
+  background: url("../images/twt.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+}
+
+#g {
+  background: url("../images/g+.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+  margin-left: 23px;
+}
+
+.newsletter {
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
   text-align: center;
 }
 
-a {
-  color: rgba(#000, 0.7);
+footer .box-social {
+  align-content: center;
+  width: 350px;
+}
 
-  &:focus,
-  &:hover {
-    color: rgba(#000, 0.6);
-  }
+footer .box-social button:hover {
+  cursor: pointer;
+}
+
+footer .contenedorsocial {
+  align-items: center;
+  display: inline-flex;
+  justify-content: center;
+  width: 66%;
+}
+
+footer .contenedorsocial .logo {
+  margin-left: 96px;
+  margin-right: 96px;
+}
+
+footer .box-social p {
+  text-align: center;
+}
+
+footer .contenedornews {
+  width: 33%;
+  display: inline-flex;
+}
+
+footer .newsletter input {
+  height: 35px;
+  width: 300px;
+  justify-content: center;
+}
+
+footer .information ul li a {
+  height: 22px;
+  width: 192px;
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+  text-decoration: none;
+  align-items: center;
+  margin-right: 50;
+}
+
+footer .information ul li {
+  display: inline-block;
+}
+
+footer .information ul {
```

Comment:
Merge rule `footer .information ul` with rule on line 2<br>Selector should have depth of applicability no greater than 2, but was 3